src/main.rs

```diff
@@ -61,7 +61,7 @@ struct Args {
     #[clap(long = "host-seed", env = "WASMCLOUD_HOST_SEED")]
     host_seed: Option<String>,
     /// Delay, in milliseconds, between requesting a provider shut down and forcibly terminating its process
-    #[clap(long = "provider-shutdown-delay", default_value = "300", env = "WASMCLOUD_PROV_SHUTDOWN_DELAY_MS", value_parser = parse_duration)]
+    #[clap(long = "provider-shutdown-delay", default_value = "300", env = "WASMCLOUD_PROV_SHUTDOWN_DELAY_MS", value_parser = parse_duration_millis)]
     provider_shutdown_delay: Duration,
```

Comment:
can we have this be `provider-shutdown-delay-ms`? an alias should work right?

Reply:
I think that would make sense for consistency, happy to adjust 👍 

I do think an alias should work